### PR TITLE
new pagerduty cases

### PR DESCRIPTION
**Pull request by trriplejay**

### CODE REVIEW

File: tests/yaml/S_PS_I_pagerduty_0009.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    steps:
+      - name: S_PS_I_pagerduty_0009
+        type: PowerShell
+        configuration:
+          nodePool: win_2019
+          integrations:
+            - name: i_pagerduty
+        execution:
+          onExecute:
+            - send_notification i_pagerduty --text "hello world from ps run number $run_number"
```
Comment (raksha78) [inline tests/yaml/S_PS_I_pagerduty_0009.yml:12]: According to the test case i think -text should have single dash

File: tests/yaml/S_PS_I_pagerduty_0007.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+ pipelines:
+  - name: pipelines_S_PS_I_pagerduty_0007
+    steps:
+      - name: S_PS_I_pagerduty_0007
+        type: PowerShell
+        configuration:
+          nodePool: win_2019
+          integrations:
+            - name: PSSI_pagerduty
```
Comment (raksha78) [inline tests/yaml/S_PS_I_pagerduty_0007.yml:9]: Why are we using the same integration for checking for invalid key and invalid length?

File: tests/yaml/S_PS_I_pagerduty_0009.yml (hunk 1 above)
Comment (trriplejay) [reply tests/yaml/S_PS_I_pagerduty_0009.yml:12]: I think the description is wrong. i'll update it. It was copied from bash. in bash the test is to try a single dash, in PS the test is to try two.

Comment (raksha78) [reply tests/yaml/S_PS_I_pagerduty_0009.yml:12]: Okay

**Pull request merged**